### ISSUE TITLE
Doesn't seem to reach discord?

### ISSUE THREAD

**Issue by mavermc** [irbyjm/trakt-discord-presence#13]
I tried destroying the discord app and rebuilding it, still the program seems to run fine and properly is pulling info from trakt, but it never ends up changing status on discord. Was working last year but I haven't touched it for a few months. Is there a way I can have it output a log of the discord side of the script? So I can try to narrow down what the issue is? no errors on startup either.

**Comment by irbyjm** [comment]
In lib/rpc.py, change
`logger = logging.getLogger(__name__)`
to
`logger = logging.getLogger(__name__)                                                                                                                                         
logging.basicConfig(filename='app.log',level=logging.DEBUG)`

This should dump your Trakt and Discord information into a file; look for `DEBUG:lib.rpc` events for your Discord send/receive status--if it worked, you should see `INFO:lib.rpc: connected via ID`.

**Comment by mavermc** [comment]
It looks like it connects just fine, I have it running on a computer with discord desktop opened up. Is there any switches I should double check on the discord side. it feels like I'm missing something stupid simple. I posted a sterilized log file. I couldn't see anything amiss but I'm a little out of element at this point. I know nothing of how the discord API works. 

[app.log](https://github.com/irbyjm/trakt-discord-presence/files/4533942/app.log)

**Comment by mavermc** [comment]
It looks like it connects just fine, I have it running on a computer with discord desktop opened up. Is there any switches I should double check on the discord side. it feels like I'm missing something stupid simple. I posted a sterilized log file. I couldn't see anything amiss but I'm a little out of element at this point. I know nothing of how the discord API works.

[app.log](https://github.com/irbyjm/trakt-discord-presence/files/4533948/app.log)

**Comment by irbyjm** [comment]
Yeah, it looks solid. You're finding a pid and sending the watch status... Is this the newest version of the Discord client? When looking into your issue, the first time I restarted my script it sent watch info but the timer was stuck at 00:00. Then when I restarted Discord, it took at least five minutes for the script to find RPC again.

**Comment by mavermc** [comment]
Yeah I tried to do a reinstall of discord, reboot the machine. Sucks this was a fresh install of windows 10. I guess I'll try destroying and remaking the discord app.

**Comment by mavermc** [comment]
Is it possible this could fail if the images aren't properly setup?

Under discord applications, rich presence, where you upload assets, I can pick a name for them, but I can't set them are small or large image (and I feel like I used to be able to)

**Comment by irbyjm** [comment]
Fingers crossed! I don't normally run this under Windows; but just set everything up and it's working for me. Here's my Discord client info:
```
Stable 58583 (b7763a5)
Host 0.0.306
Windows 10 64-Bit (10.0.18362)
```

It looks like Large/Small get set under `Rich Presence > Visualizer`, I just noticed mine weren't set properly anymore. Even after removing half of my images from the Discord app, my rich presence is updating the way I expect.
